shared/components/BaseButton/BaseButton.modules.scss

```diff
@@ -25,19 +25,19 @@ $button-text-color: $color-white;
   transition: background 0.2s;
 }
 
-.fullWidth {
+.buttonSizing {
```

Comment:
Small comment for the future when using CSS modules. You are already in a file called `BaseButton`, so all class names will be automatically scoped by that. So there is no need to use the word "button" in your class name, as that is implied already. Naming this class just "sizing" would be appropriate.